Hardware/HardwareTests/ReceiptRendererTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ReceiptRendererTests.swift
+//  HardwareTests
+//
+//  Created by Allen Snook on 6/16/21.
+//
+
+import XCTest
+
+class ReceiptRendererTests: XCTestCase {
```

Comment:
It looks like we don't actually need these tests?

Reply:
Oops! My bad. Yeah, I started building them out and then realized that we 1) don't have a Swifty way to parse HTML and 2) plan on removing the receipt implementation if we can anyhow. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4441/commits/3d5c24f4bc76a4f4862ddf9aa23b7021bca4928e